src/content/docs/apm/agents/net-agent/other-installation/understanding-net-agent-environment-variables.mdx

```diff
@@ -62,7 +62,7 @@ CORECLR_PROFILER={36032161-FFC0-4B61-B559-F6C5D41BAE5A}
 CORECLR_PROFILER_PATH="C:\Program Files\New Relic\.NET Agent\netcore\NewRelic.Profiler.dll"
 ```
 
-Note: There is a subtle difference in the behavior of the .NET framework agent vs. the .NET Core agent regarding having profiling enabled. For the .NET framework agent, even when profiling is enabled in the environment of a .NET framework process, it will only be instrumented by the agent if one of the following conditions are met:
+Note: There is a subtle difference in the behavior of the .NET Framework agent vs. the .NET Core agent regarding having profiling enabled. For the .NET Framework agent, even when profiling is enabled in the environment of a .NET Framework process, it will only be instrumented by the agent if one of the following conditions are met:
```

Comment:
```suggestion
Note: The behavior of .NET agent profiling differs slightly between the .NET Framework and .NET Core. For .NET Framework, even when profiling is enabled in the environment of a .NET Framework process, it will only be instrumented by the agent if one of the following conditions are met:
```